# Patch to fix likeback_* icons install condition. Obtained for upstream's svn
# revision 5667.
--- kmess.orig/data/icons/CMakeLists.txt	2009/07/14 21:31:05	5063
+++ kmess/data/icons/CMakeLists.txt	2009/11/08 21:09:03	5667
@@ -3,12 +3,12 @@
 
 SET( KMESS_ICON_INSTALL_DIR ${DATA_INSTALL_DIR}/kmess/icons/hicolor/16x16 )
 
-# If we're compiling in debug mode, also add the LikeBack icons
-IF( ${KMESS_ENABLE_DEBUG_OUTPUT} EQUAL 1 )
+# If Likeback is enabled, install the needed icons
+IF( NOT ${KMESS_DISABLE_LIKEBACK} EQUAL 1 )
   INSTALL( FILES likeback_bug.png     DESTINATION ${KMESS_ICON_INSTALL_DIR}/actions )
   INSTALL( FILES likeback_dislike.png DESTINATION ${KMESS_ICON_INSTALL_DIR}/actions )
   INSTALL( FILES likeback_feature.png DESTINATION ${KMESS_ICON_INSTALL_DIR}/actions )
   INSTALL( FILES likeback_like.png    DESTINATION ${KMESS_ICON_INSTALL_DIR}/actions )
-ENDIF( ${KMESS_ENABLE_DEBUG_OUTPUT} EQUAL 1 )
+ENDIF( NOT ${KMESS_DISABLE_LIKEBACK} EQUAL 1 )
 
 INSTALL( FILES kmess-shadow.png DESTINATION ${KMESS_ICON_INSTALL_DIR}/apps )
